sql/src/main/java/io/crate/operation/collect/StatsTables.java

```diff
@@ -49,12 +50,13 @@
  * in the same jvm the memoryTables aren't shared between the nodes.
  */
 @ThreadSafe
+@Singleton
 public class StatsTables {
 
     protected final Map<UUID, JobContext> jobsTable = new ConcurrentHashMap<>();
     protected final Map<Integer, OperationContext> operationsTable = new ConcurrentHashMap<>();
-    protected final AtomicReference<BlockingQueue<JobContextLog>> jobsLog = new AtomicReference<>();
-    protected final AtomicReference<BlockingQueue<OperationContextLog>> operationsLog = new AtomicReference<>();
+    protected final AtomicReference<Queue<JobContextLog>> jobsLog = new AtomicReference<>();
```

Comment:
initialize with NOOP_QUEUES right here, so it's clearer that it should never be null